dotCMS/src/main/java/com/dotcms/vanityurl/business/VanityUrlAPI.java

```diff
@@ -0,0 +1,76 @@
+package com.dotcms.vanityurl.business;
+
+import com.dotcms.vanityurl.model.CachedVanityUrl;
+import com.dotcms.vanityurl.model.VanityUrl;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.liferay.portal.model.User;
+import java.util.List;
+
+/**
+ * This API provides access to the information related to Vanity URLs
+ * in dotCMS. Vanity URLs are alternate reference paths to
+ * internal or external URL's. Vanity URLs are most commonly used to give
+ * visitors to the website a more user-friendly or memorable way of reaching an
+ * HTML page or File, that might actually live “buried” in a much deeper path.
+ *
+ * @author oswaldogallango
+ * @version 4.2.0
+ * @since June 12, 2017
+ */
+public interface VanityUrlAPI {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)